react/components/FilterOptionTemplate.js

```diff
@@ -129,7 +129,7 @@ const FilterOptionTemplate = ({
 
     const shouldTruncate =
       (truncateFilters || isLazyFacetsFetchEnabled) &&
-      quantity > FACETS_RENDER_THRESHOLD
+      quantity > FACETS_RENDER_THRESHOLD + 1
```

Comment:
Why this +1?

Reply:
Just to avoid truncating a single value. I added a comment to this line to make it easier to understand.

Quoting from http://baymard.com/

> Never truncate a single value. It simply doesn’t make sense to truncate a single value since showing the truncation link will take up the same space as just showing that value. It’s only when 2+ options are truncated that the desired space efficiencies can be realized and thus justify the truncation interface. Programmatically, this can be achieved by simply offsetting the truncation logic by one, so that the truncation threshold is actually 2 items off the filter value size that invokes a truncation interface.